src/server/api.php

```diff
@@ -2028,6 +2092,30 @@ function meta_delphi() {
       $epidata = get_covid_hosp_facility_lookup($state, $ccn, $city, $zip, $fips_code);
       store_result($data, $epidata);
     }
+  } else if($source === 'covidcast_nowcast') {
+    if(require_all($data, array('data_source', 'time_type', 'geo_type', 'time_values'))
+       && require_any($data, array('signal', 'signals'))
```

Comment:
i mean I guess you could do this as accepting either `signal` or `signals` just to be nice, but it's up to you, there's arguably no existing user base to consider
```suggestion
    if(require_all($data, array('data_source', 'time_type', 'geo_type', 'time_values', 'signals'))
```